<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ODKGeo_QgisWktDialogBase</class>
 <widget class="QDialog" name="ODKGeo_QgisWktDialogBase">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>230</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>ODK Geo to QGIS WKT</string>
  </property>
  
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0" colspan="2">
    <widget class="QLabel" name="labelFile">
     <property name="text">
      <string>Select ODK Data File (.xlsx):</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QgsFileWidget" name="xlsFileWidget">
     <property name="toolTip">
      <string>Select the .xls file</string>
     </property>
    </widget>
   </item>

   <item row="1" column="0">
    <widget class="QLabel" name="labelSheet">
     <property name="text">
      <string>Select Sheet:</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QComboBox" name="sheetDropdown"/>
   </item>

   <item row="2" column="0" colspan="2">
    <widget class="QCheckBox" name="autoSelectCheckbox">
     <property name="text">
      <string>Check to auto-select MAHSA ODK Geometry Columns</string>
     </property>
    </widget>
   </item>

   
   <item row="4" column="0">
    <widget class="QLabel" name="labelTrace">
     <property name="text">
      <string>Select ODK line geometry data column:</string>
     </property>
    </widget>
   </item>
   <item row="4" column="1">
    <widget class="QComboBox" name="traceColumnDropdown"/>
   </item>

   <item row="5" column="0">
    <widget class="QLabel" name="labelTraceResult">
     <property name="text">
      <string>Enter name for converted line geometry column:</string>
     </property>
    </widget>
   </item>
   <item row="5" column="1">
    <widget class="QLineEdit" name="traceResultColumnName">
     <property name="placeholderText">
      <string>Enter custom column name</string>
     </property>
    </widget>
   </item>

   <item row="7" column="0">
    <widget class="QLabel" name="labelPolygon">
     <property name="text">
      <string>Select ODK polygon geometry data column:</string>
     </property>
    </widget>
   </item>
   <item row="7" column="1">
    <widget class="QComboBox" name="polygonColumnDropdown"/>
   </item>

   <item row="8" column="0">
    <widget class="QLabel" name="polyTraceResult">
     <property name="text">
      <string>Enter name for converted polygon geometry column:</string>
     </property>
    </widget>
   </item>
   <item row="8" column="1">
    <widget class="QLineEdit" name="polyResultColumnName">
     <property name="placeholderText">
      <string>Enter custom column name</string>
     </property>
    </widget>
   </item>

   <item row="9" column="1" alignment="Qt::AlignRight">
    <widget class="QPushButton" name="convertButton">
     <property name="toolTip">
      <string>Click to convert coordinates to WKT</string>
     </property>
     <property name="text">
      <string>Convert</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>

 <customwidgets>
  <customwidget>
   <class>QgsFileWidget</class>
   <extends>QWidget</extends>
   <header>qgsfilewidget.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
